README.md

```diff
@@ -54,3 +54,17 @@ Add 2 input fields with placeholders ‘a’ and ‘b’ accordingly.
 Add button [Hypotenuse](should calculate hypotenuse from values, provided in the inputs and output result in an  ‘alert’ window). 
 - Section 5. Pick a palette, make picture of it, put it in the same directory as the HTML file and display it in this section of the page. Add ‘Palette’ title to the image
 Style your page (including background color and text color) with colors of palette you’ve picked
+
+## Homework3
+1. Create branch ‘<username>-hw3’. Make the following changes:
+    1. Update 'README.md' with task of this homework
+    2. Implement a HTML page with table of TOP 5 favorite flowers. Columns: 
+        - № (just an index number from 1 to 5)
+        - Name
+        - Picture (store pictures locally, - create a folder called ‘assets’)
+        - Description (a short description of the flower)
+    3. Make a ‘zebra’ view of the table
+    4. Add a CSS style that changes background color of a table row’s on hover
+    5. Use palette when picking colors
```

Comment:
Please, implement this point as well. 
You may pick any palette and set some custom colors to, for example:
- table's border 
- text
- table's zebra